packages/checkbox/README.md

```diff
@@ -1,16 +1,14 @@
 # [`@fremtind/jkl-checkbox`](https://fremtind.github.io/jokul/components/checkbox/)
 
-## Om avmerkingsbokser
+# Avmerkingsbokser
+En gruppe avmerkingsbokser brukes til å gi en liste over valg der brukeren kan velge fritt blant alternativene. Enkeltstående avmerkingsbokser kan brukes for å innhente et samtykke i en prosess. Ledeteksten på hver avmerkingsboks beskriver alternativet.
```

Comment:
Synes fortsatt det bør stå "et eller flere alternativer" eller noe lignende: Du kan velge fritt blandt alternativene med radioknapper også.

Reply:
Forslag: 
Avmerkingsbokser kan brukes enkeltstående eller som en liste med flere alternativer. 
Enkeltstående avmerkingsbokser brukes typisk for å innhente aksept mens avmerkingsbokser i listeform gir brukeren mulighet til å velge flere alternativer. Der man bare kan velge ett alternativ brukes radiobuttons.
Avmerkingsbokser har alltid en ledetekst som beskriver alternativet.